### PR TITLE
Add some GC papers

### PR DESCRIPTION
**Pull request by jacob-hughes**

### CODE REVIEW

File: softdev.bib [hunk 1]
```diff
@@ -2757,3 +2757,24 @@ @misc{watson21assessing
     month = Sep,
     url = "https://www.capabilitieslimited.co.uk/pdfs/20210917-capltd-cheri-desktop-report-version1-FINAL.pdf"
 }
+
+@article{degenbaev18cross,
+    title = "Cross-component garbage collection",
+    author = "Degenbaev, Ulan and Eisinger, Jochen and Hara, Kentaro and Hlopko, Marcel and Lippautz, Michael and Payer, Hannes",
+    year = 2018,
+    month = Oct,
+    booktitle = "OOPSLA",
```
Comment (ltratt) [inline softdev.bib:2766]: There should be `pages` for this I think?

Comment (jacob-hughes) [reply softdev.bib:2766]: Added here: 69bab97d48c2da68f3012fb4e857490431cea217

**Comment by ltratt** [comment]
Please squash (note there are conflicts so you'll have to rebase against master too).

**Comment by jacob-hughes** [comment]
Squashed

**Pull request merged**